vispy/visuals/tests/test_ellipse.py

```diff
@@ -19,20 +19,20 @@ def test_circle_draw():
         ellipse = visuals.Ellipse(pos=(75, 35, 0), radius=20,
                                   color=(1, 0, 0, 1))
         c.draw_visual(ellipse)
-        assert_image_equal("screenshot", 'visuals/circle1.png')
+        assert_image_approved("screenshot", 'visuals/circle1.png')
```

Comment:
Do we need all three of `assert_image_equal`, `assert_image_approved` and `assert_image_match`? Can we eliminate one of them? I think match uses pixel-by-pixel matching whereas equal uses maximal cross-correlation value, and it looks like approved just maps to match. I can see why we'd want cross-corr in some cases and pixel differences in another, but can we combine approved and match? Or make it so approved can have a mode where you choose which comparison to use?